src/utils.py

```diff
@@ -17,10 +19,10 @@
 
 
 RowType = Literal["row", "database", "workspace"]
```

Comment:
Can we choose a more general term than "row" such as "object". I think its confusing to overload "row".

Reply:
fixed